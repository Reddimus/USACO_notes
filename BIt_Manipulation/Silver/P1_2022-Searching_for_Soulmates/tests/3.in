10
82290 63957
10791 72602
36069 25985
6219 21855
56556 36740
15886 27608
20427 13280
92061 27932
10715 70949
89108 76962
