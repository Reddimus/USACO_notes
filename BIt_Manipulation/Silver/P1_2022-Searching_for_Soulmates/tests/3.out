44
41
36
39
35
39
32
35
37
43
